src/screens/HyperSwitch/Connectors/ConnectPayPalFlow/ConnectPayPal.res

```diff
@@ -0,0 +1,511 @@
+let h3Leading2TextClass = `${HSwitchUtils.getTextClass(
+    ~textVariant=H3,
+    ~h3TextVariant=Leading_2,
+    (),
+  )} text-grey-700`
+let p1RegularTextClass = `${HSwitchUtils.getTextClass(
+    ~textVariant=P1,
+    ~paragraphTextVariant=Regular,
+    (),
+  )} text-grey-700 opacity-50`
+
+let p1MediumTextClass = `${HSwitchUtils.getTextClass(
+    ~textVariant=P1,
+    ~paragraphTextVariant=Medium,
+    (),
+  )} text-grey-700`
+let p2RedularTextClass = `${HSwitchUtils.getTextClass(
+    ~textVariant=P2,
+    ~paragraphTextVariant=Regular,
+    (),
+  )} text-grey-700 opacity-50`
+
+let preRequisiteList = [
+  "You need to grant all the permissions to create and receive payments",
+  "Confirm your email id once PayPal sends you the mail",
+]
+
+module PayPalCreateNewAccountModal = {
+  @react.component
+  let make = (~butttonDisplayText, ~actionUrl, ~setScreenState) => {
+    let initializePayPalWindow = () => {
+      try {
+        Window.payPalCreateAccountWindow()
+      } catch {
+      | Js.Exn.Error(e) =>
+        switch Js.Exn.message(e) {
+        | Some(message) => setScreenState(_ => PageLoaderWrapper.Error(message))
+        | None => setScreenState(_ => PageLoaderWrapper.Error("Failed to load paypal window!"))
+        }
+      }
+    }
+
+    React.useEffect0(() => {
+      initializePayPalWindow()
+      None
+    })
+
+    <button
```

Comment:
use the <Button Component

Reply:
we have to use core button component as we have to pass children and Button component doesn't allow